frontend/src/components/EditRecipe/EditRecipeForm.js

```diff
@@ -1,86 +1,85 @@
-import React, { Component } from 'react';
-
+import React, { useState } from 'react';
+import { createRecipe } from '../../lib/apiClient';
 import Footer from '../Footer/Footer';
 import './EditRecipeForm.css';
 
-class EditRecipeForm extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      title: '', // eslint-disable-line react/no-unused-state
-      snippet: '', // eslint-disable-line react/no-unused-state
-    };
-    this.handleInputChange = this.handleInputChange.bind(this);
-  }
+const EditRecipeForm = ({ user }) => {
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [difficulty, setDifficulty] = useState('Advanced');
+  const [duration, setDuration] = useState('60 mins');
+  const [servings, setServings] = useState('2');
 
-  handleInputChange(event) {
-    this.setState({
-      [event.target.name]: event.target.value,
+  const handleSubmit = () => {
+    const currentUserId = user.id;
+    createRecipe({
+      title, description, difficulty, duration, servings, currentUserId,
     });
```

Comment:
I think this method should give the user some sort of confirmation when their recipe has been submitted. Maybe the create recipe boxes disappear and are replaced with a message saying "Recipe Created!"

Reply:
Yeah, I think ideally we actually push the user to the created recipe page.